hugr-model/src/v0/mod.rs

```diff
@@ -4,43 +4,38 @@
 //! It is included in the library to allow for early experimentation, and for
 //! the core and model to converge incrementally.
 //!
-//! # Terms
+//! This module defines representations of the hugr IR as plain data, designed
+//! to be as independent of implementation details as feasible. It can be used
+//! by the core compiler, alternative implementations or tooling that does not
+//! need the power/complexity of the full compiler. We provide the following
+//! representations, including conversions between them:
 //!
-//! Terms form a meta language that is used to describe types, parameters and metadata that
-//! are known statically. To allow types to be parameterized by values, types and values
-//! are treated uniformly as terms, enabling a restricted form of dependent typing.
-//! The type system is extensible and can be used to declaratively encode the desired shape
-//! of operation parameters and metadata. Type constraints can be used to express more complex
-//! validation rules.
-//!
-//! # Tabling
-//!
-//! Instead of directly nesting structures, we store them in tables and refer to them
-//! by their index in the table. This allows us to attach additional data to the structures
-//! without changing the data structure itself. This can be used, for example, to keep track
-//! of metadata that has been parsed from its generic representation as a term into a more
-//! specific in-memory representation.
-//!
-//! The tabling is also used for deduplication of terms. In practice, many terms will share
-//! the same subterms, and we can save memory and validation time by storing them only once.
-//! However we allow non-deduplicated terms for cases in which terms carry additional identity
-//! over just their structure. For instance, structurally identical terms could originate
-//! from different locations in a text file and therefore should be treated differently when
-//! locating type errors.
+//! - [Table]: Efficient intermediate data structure to facilitate conversions.
+//! - [Binary]: Binary serialisation format optimised for performance.
+//! - [AST]: Abstract syntax tree that is more ergonomic than the table form.
+//! - [Text]: Human readable s-expression based text format.
```

Comment:
Hang on. This is a reference to exactly the same thing as the previous `[AST]`

Reply:
Yes:
```suggestion
//! - [AST]: Human readable s-expression based text format.
```
and I think we're good